config.md

```diff
@@ -85,7 +85,7 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 ## Process configuration
 
-* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
+* **`terminal`** (bool, optional) specifies whether a terminal is allocated for and attached to that process. Defaults to false.
```

Comment:
On Wed, Jul 20, 2016 at 12:10:26PM -0700, Mrunal Patel wrote:
> > -* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
> > +* **`terminal`** (bool, optional) specifies whether a terminal is allocated for and attached to that process. Defaults to false.
>
> @wking I think that both approaches are valid and we shouldn't add
> language to support one but not the other.

I'm ok with being vague in config.md as long as it gets spelled out in
the command-line API specs.  Otherwise this is going to be hard to
compliance-test.  If we're trying to focus just on the
inside-the-container view here, how about:

  If true, the container process's standard streams will all be
  attached to a terminal.  Defaults to false.

If we're not going into detail about who allocates the terminal, I'd
rather not mention “allocate” at all.